support-logs/retention-lambda/lambda_function.py

```diff
@@ -0,0 +1,33 @@
+import boto3
+from requests_aws4auth import AWS4Auth
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+import curator
+
+host = 'vpc-support-elk-domain-vcesheatvi6xspmp4sd6dnfkq4.eu-west-1.es.amazonaws.com'
+region = 'eu-west-1'
+service = 'es'
+credentials = boto3.Session().get_credentials()
+awsauth = AWS4Auth(credentials.access_key, credentials.secret_key, region, service, session_token=credentials.token)
+
+# Lambda execution starts here.
+def lambda_handler(event, context):
+
+    # Build the Elasticsearch client.
+    es = Elasticsearch(
+        hosts = [{'host': host, 'port': 443}],
+        http_auth = awsauth,
+        use_ssl = True,
+        verify_certs = True,
+        connection_class = RequestsHttpConnection
+    )
+
+    index_list = curator.IndexList(es)
+
+    # Filters by age, anything with a time stamp older than 30 days in the index name.
```

Comment:
Can we bump this to 3 months to start off with?

Reply:
I think that's OK for the logs that are currently being streamed. I set it to something short for testing purposes!